k8s-operator/apis/v1alpha1/types_tsdnsconfig.go

```diff
@@ -85,6 +86,9 @@ type Nameserver struct {
 	// Service configuration.
 	// +optional
 	Service *NameserverService `json:"service,omitempty"`
+	// If specified, the nameserver Pod's tolerations.
+	// +optional
+	PodTolerations []v1.Toleration `json:"podTolerations,omitempty"`
```

Comment:
Should this go under `.spec.nameserver.deployment.pod`, similar to ProxyClass? The only wrinkle is we already have `.spec.nameserver.image` which is a bit inconsistent.

Reply:
I agree here that maybe this isn't the right level. Frustrating that we have `.spec.nameserver.image` though for sure, there is definitely inconsistency around this across our CRDs.